tests/testthat/test-relpath.R

```diff
@@ -0,0 +1,39 @@
+test_dir <- createRepo()
+withr::local_dir(test_dir)
+
+# Add path test
+system("mkdir script")
+system("mkdir data")
+add_file("script/file.txt", "something")
+add_file("data/filespec.txt", "something")
+logCreate()
+
+# Confirm users are able to provide a path with ../ or give an absolute path
+test_that("relpath outputs script paths relative to the QC log location: for unresolved or absolute path [REV-REL-001]", {
+  expect_identical("script/file.txt", pathFromLogRoot("data/../script/file.txt"))
+  expect_identical("script/file.txt", relPath("data/../script/file.txt"))
+  expect_identical("script/file.txt", pathFromLogRoot("/tmp/svn-testing/test/script/file.txt"))
+})
+
+logAssign("script/file.txt")
+logAssign("data/../script/file.txt")
+logAssign("script/../data/filespec.txt")
+add_commit("first")
+qclog <- readr::read_csv("QClog.csv")
+
+test_that("relpath outputs script paths relative to the QC log location - only resolved file paths are stored in the QClog [REV-REL-001]", {
+  expect_true(length(unique(qclog$file)) == 2)
```

Comment:
This might be overkill, but I think it's worth explicitly checking that these paths are actually what we expect them to be. Replace this with something like

```r
expect_identical(nrow(qclog), 3)
expect_identical(
  unique(qclog$file), 
  c("script/file.txt", "data/filespec.txt")
)
```

Reply:
Makes sense to me, just updated it